packages/graph-editor/src/components/flow/wrapper/base.tsx

```diff
@@ -91,11 +90,9 @@ export const BaseNodeWrapper = (props: NodeProps) => {
             >
               <Stack direction="row" gap={2} align="center">
                 {icon && (
-                  <IconoirProvider
-                    iconProps={{ width: '0.875em', height: '0.875em' }}
-                  >
+                  <div style={{ width: '0.875em', height: '0.875em' }}>
```

Comment:
Does the icon expand correctly into this ? I don't think it has an implicit height & width of 100%

Reply:
How do I ensure this?

Reply:
fyi you can  just pass `width` and `height` into the icon which is the preferred way of scaling the icon as then the stroke will also adjust